core/src/components/checkbox/checkbox.ionic.scss

```diff
@@ -201,7 +201,13 @@ input {
   --focus-ring-color: #{globals.$ion-border-focus-error};
 
   .native-wrapper {
```

Comment:
This should only be applied when the checkbox isn't checked, according to the Figma. It should work similarly to the way you change it for disabled checkboxes below.

Reply:
done